integrationtest/pg/pg_test.go

```diff
@@ -0,0 +1,264 @@
+package pg
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/multiformats/go-multiaddr"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	tc "github.com/textileio/go-threads/api/client"
+	"github.com/textileio/go-threads/core/thread"
+	tutil "github.com/textileio/go-threads/util"
+	"github.com/textileio/powergate/api/client"
+	"github.com/textileio/powergate/health"
+	"github.com/textileio/textile/api/apitest"
+	c "github.com/textileio/textile/api/buckets/client"
+	buckets_pb "github.com/textileio/textile/api/buckets/pb"
+	"github.com/textileio/textile/api/common"
+	hc "github.com/textileio/textile/api/hub/client"
+	"github.com/textileio/textile/util"
+	"google.golang.org/grpc"
+)
+
+var powMultiaddr = multiaddr.StringCast("/ip4/127.0.0.1/tcp/5002")
+
+func TestCreateBucket(t *testing.T) {
+	powc := spinup(t)
+	ctx, client := setup(t)
+
+	lst, err := powc.FFS.ListAPI(ctx)
+	require.NoError(t, err)
+	require.Equal(t, 0, len(lst))
+
+	_, err = client.Init(ctx, "bucky")
+	require.Nil(t, err)
+
+	lst, err = powc.FFS.ListAPI(ctx)
+	require.NoError(t, err)
+	require.Equal(t, 1, len(lst))
+}
+
+func TestArchiveBucketWorkflow(t *testing.T) {
```

Comment:
Might be good to look at this test.
Kind of covering the usual workflow.

Reply:
So much going on under the hood, this makes it look too easy 👍